pkgs/applications/science/logic/abella/default.nix

```
@@ -1,11 +1,11 @@
 { lib, stdenv, fetchurl, rsync, ocamlPackages }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "abella";
   version = "2.0.8";
 
   src = fetchurl {
-    url = "http://abella-prover.org/distributions/${pname}-${version}.tar.gz";
+    url = "http://abella-prover.org/distributions/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

pname should not be used like this
```suggestion
    url = "http://abella-prover.org/distributions/abella-${finalAttrs.version}.tar.gz";
```

---

Inlined pname.